### PR TITLE
Entity/recipe/applying builder design pattern

### LINKED ISSUE #52: Modify Recipe entity class to builder design
Modified Recipe entity class to builder design to enhance flexibility.

### PR DESCRIPTION
**Pull request by seonghyunban**
Applied builder design pattern to simplify the creation of recipe.
Later, when implementing users create a custom recipe, builder design pattern will allow sequential configuration of recipe.
Resolves #52

### CODE REVIEW

File: src/main/java/entity/Recipe.java
Comment (hhpfiona) [inline src/main/java/entity/Recipe.java:1]: nice, builder design. should we try to implement this elsewhere for other entities too?

**Review by hhpfiona — APPROVED** [review]
pushed some target files too. honestly intellij keeps making me push those chases as well even when i put them into git ignore. not sure what solution could be. otherwise nice job.

File: .gitignore [hunk 1]
```diff
@@ -14,7 +14,7 @@ src/main/java/.DS_Store
 =======
 # Output
 out
-target
+/target/*
```
Comment (parkhangorodsky) [inline .gitignore:17]: does this make it work?

**Review by parkhangorodsky — APPROVED** [review]
looks good to me! great work.

**Comment by parkhangorodsky** [comment]
please link an issue

**Pull request merged**